plugins/arRestApiPlugin/config/arRestApiPluginConfiguration.class.php

```diff
@@ -24,7 +24,7 @@ class arRestApiPluginConfiguration extends sfPluginConfiguration
 {
     public const REGEX_UUID = '[0-9a-f]{8}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{4}-[0-9a-f]{12}';
     public const REGEX_ID = '\d+';
-    public const REGEX_SLUG = '[0-9A-Za-z-]+';
+    public const REGEX_SLUG = '[0-9A-Za-z-_~:;,=*@]+';
```

Comment:
@melaniekung This looks good, but I'm curious how you determined which characters to add to the Regex?

Reply:
Ah, I think the new characters correspond to the "extraSet" values defined in: https://github.com/artefactual/atom/blob/qa/2.x/lib/model/QubitSlug.php#L73

Reply:
I think the regex for valid slug character should be defined once in the `QubitSlug` class (via a static method like `validSlugRegex()` or similar?) and then called here, rather than maintaining two different regex strings in two different parts of the code.